messenger.rst

```diff
@@ -865,6 +865,108 @@ If the message fails again, it will be re-sent back to the failure transport
 due to the normal :ref:`retry rules <messenger-retries-failures>`. Once the max
 retry has been hit, the message will be discarded permanently.
 
+Multiple failed transports
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sometimes it is not enough to have a single, global ``failed transport`` configured because 
+some messages are more important than others. In those cases, you can override the failure transport for only specific transports:
```

Comment:
```suggestion
Sometimes it is not enough to have a single, global ``failed transport`` configured
because some messages are more important than others. In those cases, you can
override the failure transport for only specific transports:
```